onelogin/endpoints.go

```diff
@@ -0,0 +1,75 @@
+package onelogin
+
+import (
+	"fmt"
+	"net/url"
+)
+
+const (
+	usBase = "https://api.us.onelogin.com"
+	euBase = "https://api.eu.onelogin.com"
+
+	GenerateSamlAssertionPath string = "/api/1/saml_assertion"
+	GenerateTokensPath        string = "/auth/oauth2/token"
+	GetUserByEmailPath        string = "/api/1/users?email=%s"
+	VerifyFactorPath          string = "/api/1/saml_assertion/verify_factor"
+)
+
+// Endpoints represent the OneLogin API HTTP endpoints.
+type Endpoints struct {
+	Region string
+
+	base *url.URL
+}
+
+func (e *Endpoints) setBase() (err error) {
+	var base string
+	switch e.Region {
+	case "US":
+		base = usBase
+
+	case "EU":
+		base = euBase
+
+	default:
+		return fmt.Errorf("Region %q is an invalid onelogin region. Valid values are EU or US", e.Region)
```

Comment:
I would fix the capitalization to `OneLogin` here, too, for uniformity. I would also add a period at the end. Sorry for being petty about these :-)

Reply:
Consistency is always something to strive for, I'll add this now